sdk/build.gradle.kts

```diff
@@ -75,6 +75,7 @@ dependencies {
   implementation(Dependencies.kotlin)
   implementation(Dependencies.androidxCoreKtx)
   implementation(Dependencies.androidxAnnotations)
+  implementation(Dependencies.androidxRecyclerView)
```

Comment:
what's your opinion on bringing just the `DiffUtil` and `ListUpdateCallback` (not sure it would be needed) source code instead of the whole recycler view library?

This adds one more dependency we (and consumers of our library) need to worry about.

Reply:
Controversial. 
Moving will mean we have to maintain the code ourselves, check whether new fixes are available upstream every release, port them etc. 
It's also `androidx` dependency and is pretty standard in the android world, most of the apps will already contain it. But let me check how will the method count change without core shrink to make more intentional decision.

Reply:
Also, let's keep in mind that this might have an impact on users that already have recyclerview dependency but doesn't match ours.

Reply:
Ended up implementing manual algo for calculations